crmsh/bootstrap.py

```diff
@@ -1745,9 +1745,15 @@ def update_nodeid(nodeid, node=None):
                 break
         print("")
         invoke("rm -f /var/lib/heartbeat/crm/* /var/lib/pacemaker/cib/*")
-        corosync.add_node_ucast(ringXaddr_res)
-        csync2_update(corosync.conf())
-        invoke("ssh -o StrictHostKeyChecking=no root@{} corosync-cfgtool -R".format(seed_host))
+        already_have = False
+        for ip in ringXaddr_res:
```

Comment:
Python has a cleaner way to manage this kind of situation where you need to control a boolean in a loop.
It would be something like that:
```
for i in my_data:
    if my_check ==true:
        break
else:
    # if break didn't happen
    code_in_case_the_last_condition_didnt_match
```
More info here: http://book.pythontips.com/en/latest/for_-_else.html

Reply:
Thank you for suggestion!